QuizWebApp/src/main/webapp/sign-up.jsp

```diff
@@ -175,4 +205,152 @@
     }
   }
 
+
+
+  document.addEventListener("DOMContentLoaded", function () {
```

Comment:
You use some event listener to implement the "Add Child" and "Remove Child" functions.

But the code style is hard to understand, I hope you should add a default child name box, child password box and "Remove Child" button in form.

Also add the "Add Child" button, then set these buttons listener function.

I mean we should separate the UI control and JavaScript functions, don't let the event listener function do all things.